x/dualstaking/keeper/grpc_query_delegator_providers.go

```diff
@@ -33,19 +33,15 @@ func (k Keeper) DelegatorProviders(goCtx context.Context, req *types.QueryDelega
 
 	var delegations []types.Delegation
 	for _, provider := range providers {
-		indices := k.delegationFS.GetAllEntryIndicesWithPrefix(ctx, provider)
+		indices := k.delegationFS.GetAllEntryIndicesWithPrefix(ctx, provider+" "+req.Delegator)
```

Comment:
use the function to create the key

Reply:
done. see https://github.com/lavanet/lava/commit/54bfb017febb0c4111849702c621d895731b9718